testlapack/general.go

```diff
@@ -986,3 +986,29 @@ func isUpperHessenberg(h blas64.General) bool {
 	}
 	return true
 }
+
+func randomSparseGeneral(r, c, stride int, p float64, rnd *rand.Rand) blas64.General {
```

Comment:
Add a comment that marks p is the probability of a zero, and probably make some comment about it being unstructured sparse.